mobile-widgets/qml/Settings.qml

```diff
@@ -132,214 +132,204 @@ Kirigami.ScrollablePage {
-				rightPadding: Kirigami.Units.gridUnit
-				Layout.preferredWidth: gridWidth * 0.15
-			}
-			Row {
-				Layout.preferredWidth: gridWidth * 0.6
-				Rectangle {
-					id: blueRect
-					color: subsurfaceTheme.blueBackgroundColor
-					border.color: "black"
-					width: sampleRegularBlue.width + Kirigami.Units.gridUnit
-					height: Kirigami.Units.gridUnit * 2
-					Text {
-						id: sampleRegularBlue
-						text: qsTr("regular text")
-						font.pointSize: subsurfaceTheme.regularPointSize
-						color: subsurfaceTheme.blueTextColor
-						anchors {
-							horizontalCenter: parent.horizontalCenter
-							verticalCenter: parent.verticalCenter
+				TemplateLabel {
+					text: qsTr("Blue")
+					rightPadding: Kirigami.Units.gridUnit
+					Layout.preferredWidth: gridWidth * 0.15
+				}
+				Row {
+					Layout.preferredWidth: gridWidth * 0.6
+					Rectangle {
+						id: blueRect
+						color: subsurfaceTheme.blueBackgroundColor
+						border.color: "black"
+						width: sampleRegularBlue.width + Kirigami.Units.gridUnit
+						height: Kirigami.Units.gridUnit * 2
+						Text {
+							id: sampleRegularBlue
+							text: qsTr("regular text")
+							font.pointSize: subsurfaceTheme.regularPointSize
+							color: subsurfaceTheme.blueTextColor
+							anchors {
+								horizontalCenter: parent.horizontalCenter
+								verticalCenter: parent.verticalCenter
+							}
 						}
 					}
-				}
-				Rectangle {
-					color: subsurfaceTheme.bluePrimaryColor
-					border.color: "black"
-					width: sampleHighlightBlue.width + Kirigami.Units.gridUnit
-					height: Kirigami.Units.gridUnit * 2
-					Text {
-						id: sampleHighlightBlue
-						text: qsTr("Highlight")
-						font.pointSize: subsurfaceTheme.regularPointSize
-						color: subsurfaceTheme.bluePrimaryTextColor
-						anchors {
-							horizontalCenter: parent.horizontalCenter
-							verticalCenter: parent.verticalCenter
+					Rectangle {
+						color: subsurfaceTheme.bluePrimaryColor
+						border.color: "black"
+						width: sampleHighlightBlue.width + Kirigami.Units.gridUnit
+						height: Kirigami.Units.gridUnit * 2
+						Text {
+							id: sampleHighlightBlue
+							text: qsTr("Highlight")
+							font.pointSize: subsurfaceTheme.regularPointSize
+							color: subsurfaceTheme.bluePrimaryTextColor
+							anchors {
+								horizontalCenter: parent.horizontalCenter
+								verticalCenter: parent.verticalCenter
+							}
 						}
 					}
 				}
-			}
-			SsrfSwitch {
-				id: blueButton
-				Layout.preferredWidth: gridWidth * 0.25
-				checked: subsurfaceTheme.currentTheme === "Blue"
-				enabled: subsurfaceTheme.currentTheme !== "Blue"
-				onClicked: {
-					blueTheme()
-					PrefDisplay.theme = subsurfaceTheme.currentTheme
+				SsrfSwitch {
+					id: blueButton
+					Layout.preferredWidth: gridWidth * 0.25
+					checked: subsurfaceTheme.currentTheme === "Blue"
+					enabled: subsurfaceTheme.currentTheme !== "Blue"
+					onClicked: {
+						subsurfaceTheme.currentTheme = "Blue"
```

Comment:
Sorry seems something got wrong with my rebasing, locally I have corrected most of this. will force push a bit later

Reply:
no worry, this is easy to happen given how many branches you have in flight that all touch the same files.
Would it be easier to try and get the enum and theme changes sorted so we can get that merged?
I don't really want to call you additional work to change things back and forth, especially since I expect we will in the end get all of these parts merged, right?

Reply:
actually it does not depend on the WIP theming, subsurfaceTheme is the good old one.